src/App.tsx

```diff
@@ -1,7 +1,27 @@
-import React from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Header from "./components/Header";
+import About from './pages/About';
+import Home from './pages/Home';
+import Insights from './pages/Insights';
+import Container from 'react-bootstrap/Container';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-function App() {
-	return <div className="App">Hello Businext!</div>;
-}
+const App = () => {
+	return (
+		<div className="app">
+			<BrowserRouter basename={process.env.PUBLIC_URL}>
```

Comment:
Is there a proper check for this? If it doesn't exist would it just display undefined?

Reply:
It always exists, according to the create-react-app gods who said so in [a comment in the template index.html](https://github.com/businext/businext-portal/pull/1/files?diff=unified&w=1#diff-c2cc24bc9001b11b6add48a4cd8f893d5d6c6e4d1bd254158bd14ab997f552cdR16-R25)